### DISCUSSION TITLE
Export public transport route

### DISCUSSION THREAD

**Discussion by dh2-io** [osmandapp/OsmAnd#19821] [Maps Related]
Hey there 👋 
Why is it possible to export train tracks, but not possible to export routs that contain public transport/ÖPNV?

**Comment by dh2-io** [comment]
<table style="width:100%">
  <tr>
    <th>Train Track</th>
    <th>ÖPNV</th>
  </tr>
  <tr>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/96197795/868ab3df-6711-4575-95ca-7d8303605a3a"  width="300" height="auto"/> </td>
    <td><img src="https://github.com/osmandapp/OsmAnd/assets/96197795/aba0487a-4001-4466-b21e-b2dbf9b42b6f"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by DmitryAlexei** [reply]
Public transport is still in beta and does not support full navigation functionality (for example, guidance). 
https://osmand.net/docs/user/navigation/routing/public-transport-navigation#overview

**Reply by dh2-io** [reply]
I kind of guessed it this way.
Is this a planed feature?
